contrib/opencensus-ext-azure/opencensus/ext/azure/log_exporter/__init__.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2019, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import json
+import requests
+
+from opencensus.ext.azure.common import Options
+from opencensus.ext.azure.common import utils
+from opencensus.ext.azure.common.exporter import BaseExporter
+
+logger = logging.getLogger(__name__)
+
+__all__ = ['AzureExporter', 'LogHandler']
+
+
+class AzureExporter(BaseExporter):
+    """An exporter that sends logs to Microsoft Azure Monitor.
+
+    :type options: dict
+    :param options: Options for the exporter. Defaults to None.
+    """
+
+    def __init__(self, **options):
+        self.options = Options(**options)
+        if not self.options.instrumentation_key:
+            raise ValueError('The instrumentation_key is not provided.')
+        super(AzureExporter, self).__init__(**options)
+
+    def emit(self, batch, event=None):
```

Comment:
It looks like this relies on `BaseExporter` having `emit` and `export`. I thought the goal was to remove `emit` as in https://github.com/census-instrumentation/opencensus-python/pull/642#discussion_r281860719.

I don't think we should plan to move `BaseExporter` into the core library in its current state.

Reply:
That's right, we only want to keep `export`, and remove `emit`. And we also discussed that probably we don't even need a `BaseExporter` at all.

In this PR I try to focus on making the log exporter work end-to-end, without doing too much refactor.

Reply:
Problem solved for logs (traces will take more effort, probably in a separate PR).